ome/ome-zarr-py
# Download tests

To test:

 - Check travis is green (tests passing)

To check the test image creation:

```
$ python
>>> from tests.create_test_data import create_zarr
>>> create_zarr("test_data.zarr")
```

Then:
```
$ ome_zarr info test_data.zarr

$ napari test_data.zarr
```
